addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -175,12 +175,23 @@
 
         @include media-only(print) {
             font-size: 0.8rem;
+            page-break-inside: avoid;
```

Comment:
`page-break-inside` is deprecated (cf. [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/page-break-inside)), you could use [`break-inside`](https://developer.mozilla.org/en-US/docs/Web/CSS/break-inside) instead.
Also, not sure it should be wrapped in a print mediaquery.